.github/copilot-instructions.md

```diff
@@ -0,0 +1,347 @@
+term (so-called obsoletion with exact replacement), or by zero to many `consider` tags.
+
+```
+[Term]
+id: MONDO:0100229
+name: obsolete Heimler syndrome
+def: "OBSOLETE. A peroxisoome biogenesis disorder characterized by sensorineural hearing loss, enamel hypoplasia of the secondary dentition, nail abnormalities and occasional or late-onset retinal pigmentation abnormalities, in which the cause of the disease is a mutation in peroxisomal biogenesis factor 1 (PEX1) or peroxisomal biogenesis factor 6 (PEX6) genes." [MONDO:patterns/disease_series_by_gene, PMID:26387595]
+subset: ordo_disorder {source="Orphanet:3220"}
+subset: ordo_malformation_syndrome {source="Orphanet:3220"}
+synonym: "bilateral sensorineural hearing loss, enamel hypoplasia and nail defects" RELATED [GARD:0001687]
+synonym: "deafness enamel hypoplasia nail defects" RELATED [GARD:0001687]
+synonym: "deafness-enamel hypoplasia-nail defects syndrome" EXACT [MONDO:0009325]
+synonym: "Heimler syndrome" EXACT []
+synonym: "sensorineural hearing loss, enamel hypoplasia, and nail abnormalities" RELATED [GARD:0001687]
+xref: MESH:C535994 {source="MONDO:obsoleteEquivalent", source="Orphanet:3220/e", source="Orphanet:3220"}
+xref: Orphanet:3220 {source="OMIM:234580", source="MONDO:obsoleteEquivalent"}
+xref: SCTID:721085000 {source="MONDO:obsoleteEquivalent"}
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+property_value: IAO:0000231 OMO:0001000
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/3222" xsd:anyURI
+is_obsolete: true
+consider: MONDO:0100259
+```
+
+Synonyms and xrefs can be migrated judiciously,
+
+We never do complete merges now, so there should be no `alt_ids` or
+disappearing stanzas. If a user asks for a merge, they usually mean
+obsoletion with direct replacement, as here:
+
+Example:
+
+```
+[Term]
+id: MONDO:0100334
+name: obsolete viral infectious disease or sequela
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+property_value: IAO:0000231 MONDO:TermsMerged
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/pull/3118#issuecomment-871858054" xsd:anyURI
+is_obsolete: true
+replaced_by: MONDO:0100321
+```
+
+No relationship should point to an obsolete term - when you obsolete a term, you may need to also rewire
+terms to "skip" the obsoleted term.
+
+## Other metadata
+
+- Link back to the issue you are dealing with using the `term_tracker_item`
+- All terms should have definitions, with at least one definition xref, ideally a PMID
+
+
+You can sign NEW terms using with a tag-value like this: (don't tag pre-existing terms)
+
+```
+property_value: http://purl.org/dc/terms/creator doi:10.1186/s13326-024-00320-3
+```
+
+You don't need to add a date, this will be handled for you
+
+## Relationships
+
+All terms should have at least one `is_a` (this can be implicit by a logical definition, see below).
```

Comment:
I don't believe this is true as some terms only have a parent specified by a logical definition